pages/RDM_cycle/TEMPLATE_RDM_cycle_WP1.md

```diff
@@ -4,6 +4,7 @@ keywords: [Keyword 1,Keyword 2, Keyword n]
 summary: Fill in your summary or delete this property when not needed
 permalink: TEMPLATE_RDM_cycle.html
 folder: RDM_cycle
+search: false
```

Comment:
what it this "search" ? What's for?

Reply:
it is to disable that that page will end up in the search results. This line will thus need to be removed when the template is copied ...

Reply:
If you think it is not a problem the searchbar searches in the Template pages, we can leave the line out off course

Reply:
It makes sense to disable that that page from search, indeed.